pkg/installation/info.go

```diff
@@ -43,7 +43,7 @@ func getInstallationInfo(ctx context.Context, gqlClient graphql.Client) (install
 	var info installationInfo
 	err := gqlClient.ExecuteQuery(ctx, infoQuery, nil, &info)
 	if err != nil {
-		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet and that the Athena service is up and running\n%s", err.Error())
+		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet, OIDC is configured for the management cluster and that the Athena service is up and running\n%s", err.Error())
```

Comment:
```suggestion
		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet, OIDC authentication is configured for the Management API, and the Athena service is up and running\n%s", err.Error())
```

My suggestion is about adding `authentication` and replacing management cluster with `Management API`.

If that oxford comma doesn't make sense here, please fix as you think.

Reply:
Thanks Marian much better and good to add the oxford comma. 👍